src/js_parser.zig

```diff
@@ -16499,12 +16522,15 @@ fn NewParser_(
                         e_.tag = p.visitExpr(tag);
 
                         if (comptime allow_macros) {
```

Comment:
I updated the code for macro template tags but it doesn’t seem like Bun supports it.